ops/testing.py

```diff
@@ -106,6 +106,10 @@
     })
     RawConfig = TypedDict("RawConfig", {'options': Dict[str, _ConfigOption]})
 
+    _NetworkBindings = TypedDict('_NetworkBindings', {
```

Comment:
I think a typeddict is overkill for what we could simply type as `Dict[Tuple[str,Optional[int]], _NetworkDict]`.
Also I'd inline that with the definition of `Harness._networks`, I don't think there's any other object who'd be interested in that type.